pages/products.vue

```diff
@@ -0,0 +1,158 @@
+    email: 'dries.vincent@example.com',
+    category: 'Przystawka',
+    imageUrl:
+      'https://media.istockphoto.com/id/1319938305/pl/zdj%C4%99cie/pyszna-tarta-wo%C5%82owa-z-jajkiem-przepi%C3%B3rczym-i-przyprawami.jpg?s=1024x1024&w=is&k=20&c=0ET_4wQ3dNm2IgB21_e2ltXQ0wE3Svb6_x-TfMjCdDg=',
+  },
+  {
+    id: '4',
+    name: 'Zupa pomidorowa',
+    category: 'Zupa',
+    imageUrl:
+      'https://media.istockphoto.com/id/1144433097/pl/zdj%C4%99cie/zupa-pomidorowa.jpg?s=1024x1024&w=is&k=20&c=TaeDg_XRgT8NtLevdEpAjtsK6fwxaHzrxttu8YCwbDU=',
+  },
+  {
+    id: '5',
+    name: 'Beza',
+    category: 'Deser',
+    imageUrl:
+      'https://media.istockphoto.com/id/1136861359/pl/zdj%C4%99cie/widok-z-g%C3%B3ry-na-ciasto-pavlova-nowej-zelandii-z-bit%C4%85-%C5%9Bmietan%C4%85-i-mieszank%C4%85-%C5%9Bwie%C5%BCych-jag%C3%B3d-na.jpg?s=1024x1024&w=is&k=20&c=xujzYgvquBkRCXuod4NETuQ62Zi0e1vm0057n15kZDU=',
+  },
+  {
+    id: '6',
+    name: 'Aperol',
+    category: 'Alkohol',
+    imageUrl:
+      'https://media.istockphoto.com/id/1307999346/pl/zdj%C4%99cie/kieliszek-lodowego-koktajlu-aperol-spritz-podawanego-w-kieliszku-do-wina-ozdobionym.jpg?s=1024x1024&w=is&k=20&c=Noa7PMKqHx1dywt2QLIHK2PMrF5wWn0HDvpqGO9SLps=',
+  },
+  {
+    id: '7',
+    name: 'Pieczywo',
+    category: 'Dodatki',
+    imageUrl:
+      'https://media.istockphoto.com/id/1125389587/pl/zdj%C4%99cie/pyszny-domowy-chleb-%C5%BCytni-na-zakwasie-na-talerzu-i-mleku-domowe-pieczenie.jpg?s=1024x1024&w=is&k=20&c=G4AEYKt0kcMyXNSyRAenK4Tv4Zc3S4Ttv4qZXLe46jg=',
+  },
+]
+
+const suggestedProducts = [
+  {
+    id: '8',
+    name: 'Wino',
+    category: 'Napój',
+    imageUrl:
+      'https://media.istockphoto.com/id/1332101410/pl/zdj%C4%99cie/zamrozi%C4%87-ruch-czerwonego-wina-wlewaj%C4%85cego-si%C4%99-do-szk%C5%82a.jpg?s=1024x1024&w=is&k=20&c=JRzMyTimIICL6IvEqIKlB2737HIzbeXKTU8r6TZqQBQ=',
+  }
+]
+</script>
+
+<template>
+  <div class="flex w-full h-full justify-center">
+
+    <div class="m-10 flex self-center absolute z-50 border border-white rounded-lg" :class="{ hidden: isHidden }">
+      <div class="rounded-lg relative bg-whiterounded-lg shadow dark:bg-gray-900">
+
+        <div class="flex items-center justify-between p-4 md:p-5 border-b rounded-t dark:border-gray-600">
+          <h3 class="text-xl font-semibold text-gray-800 dark:text-gray-200">
+            Sugerowane produkty na podstawie Twojego wyboru
+          </h3>
+          <button @click="isHidden = true"
+            class="text-gray-800 dark:text-gray-200 hover:text-gray-800 rounded-lg text-sm w-8 h-8 ms-auto inline-flex justify-center items-center dark:hover:bg-gray-600 dark:hover:text-white">
+            <svg class="w-3 h-3" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 14 14">
+              <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+                d="m1 1 6 6m0 0 6 6M7 7l6-6M7 7l-6 6" />
+            </svg>
+          </button>
```

Comment:
Tutaj chcialas napisac wlasny css widze, ale ogolnie mozemy korzystac z NuxtUi, bo jest to podpiete i mniej roboty, a do tego komponenty sie od siebie nie beda roznic